.github/workflows/eslint-config.yml

```diff
@@ -46,4 +46,4 @@ jobs:
       - run: yarn publish
         working-directory: ${{ env.PROJECT_DIR }}
         env:
-          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_API_KEY }}
```

Comment:
I can make the old version work.

```suggestion
          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Reply:
Both work now. When this merges and I know which one you actually want to use, I'll delete the other.

Reply:
Ok, now only NPM_TOKEN works.